<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraAdvancedSettingsWidget</class>
 <widget class="QWidget" name="CameraAdvancedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QnInputField" name="cameraIdInputField" native="true"/>
   </item>
   <item>
    <widget class="QnInputField" name="primaryStreamUrlInputField" native="true"/>
   </item>
   <item>
    <widget class="QnInputField" name="secondaryStreamUrlInputField" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="noSettingsPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="noSettingsLabel">
         <property name="text">
          <string>This camera has no advanced settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="manualPage">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <item>
        <widget class="QnCameraAdvancedParamsWidget" name="cameraAdvancedParamsWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="webPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWebView" name="webView" native="true">
         <property name="url" stdset="0">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>QnCameraAdvancedParamsWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/properties/camera_advanced_params_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnInputField</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/common/input_field.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
